app/assets/v2/js/pages/bounty_details.js

```diff
@@ -662,9 +661,8 @@ var do_actions = function(result) {
     }
 
     if (show_increase_bounty) {
```

Comment:
won't you need to update `show_increase_bounty` too?

Reply:
Is it needed? what I change here is the argument from `enabled = increase_bounty_enabled;` to `enabled = true;`, the `show_increase_bounty` is fine.